opencti-platform/opencti-graphql/src/migrations/1686215994933-triggers-add-authorized-members.js

```diff
@@ -0,0 +1,27 @@
+import { executionContext, MEMBER_ACCESS_RIGHT_ADMIN, SYSTEM_USER } from '../utils/access';
+import { listAllEntities } from '../database/middleware-loader';
+import { ENTITY_TYPE_TRIGGER } from '../modules/notification/notification-types';
+import { logApp } from '../config/conf';
+import { patchAttribute } from '../database/middleware';
+
+export const up = async (next) => {
+  const context = executionContext('migration', SYSTEM_USER);
+  const triggers = await listAllEntities(context, context.user, [ENTITY_TYPE_TRIGGER]);
+  const triggersUpdates = [];
+  for (let i = 0; i < triggers.length; i += 1) {
+    const trigger = triggers[i];
+    const triggerUserIds = trigger.user_ids ?? [];
+    const authorizedMembersInput = triggerUserIds.map((userId) => {
+      return { id: userId, access_right: MEMBER_ACCESS_RIGHT_ADMIN };
+    });
+    const patch = { authorized_members: authorizedMembersInput };
+    triggersUpdates.push(patchAttribute(context, context.user, trigger.id, ENTITY_TYPE_TRIGGER, patch));
+  }
+  await Promise.all(triggersUpdates);
```

Comment:
Please use Bluebird promise instead of this, it's better in your case and will be more readable

Reply:
done